talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -150,7 +154,10 @@ void PeerConnectionDependencyFactory::
 
 #elif defined(WEBRTC_LINUX)
   // MSDK support for Linux is not in place. Use default.
-  if (encoded_frame_) {
+  if (dual_video_encoder_) {
+    RTC_LOG(LS_WARNING) << "Using DualVideoEncoder";
+    encoder_factory.reset(new DualVideoEncoder());
```

Comment:
the initial implementation is anyways just a shallow wrapper over `webrtc::CreateBuiltinVideoEncoderFactory()` below.

I will eventually update it to be a split of the built-in one (for raw frames) and what `EncodedVideoEncoderFactory()` is doing (currently unused in our flow).